sdk/videoanalyzer/Azure.Media.VideoAnalyzer.Edge/src/Customization/MediaGraphPemCertificateList.cs

```diff
@@ -0,0 +1,104 @@
+    /// </summary>
+    public partial class PemCertificateList
+    {
+        /// <summary>
+        /// How long the base 64 lines should be.
+        /// </summary>
+        private const int Base64LineLength = 76;
+
+        /// <summary>
+        /// Header for a PEM formatted certificate.
+        /// </summary>
+        /// <remarks>
+        /// See https://tools.ietf.org/html/rfc7468 Section 5.1.
+        /// </remarks>
+        private const string PemCertificateHeader = "-----BEGIN CERTIFICATE-----";
+
+        /// <summary>
+        /// Footer for a PEM formatted certificate.
+        /// </summary>
+        /// <remarks>
+        /// See https://tools.ietf.org/html/rfc7468 Section 5.1.
+        /// </remarks>
+        private const string PemCertificateFooter = "-----END CERTIFICATE-----";
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PemCertificateList"/> class.
+        /// </summary>
+        /// <param name="certificates"> The list of certificates.</param>
+        public PemCertificateList(IList<X509Certificate2> certificates)
+        {
+            if (certificates == null)
+            {
+                throw new ArgumentNullException(nameof(certificates));
+            }
+
+            Certificates = certificates.Select(ConvertToPemString).ToList();
+            Type = "#Microsoft.Media.MediaGraphPemCertificateList";
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PemCertificateList"/> class.
+        /// </summary>
+        /// <param name="certificates"> The certificates params.</param>
+        public PemCertificateList(params X509Certificate2[] certificates)
+        {
+            if (certificates == null)
+            {
+                throw new ArgumentNullException(nameof(certificates));
+            }
+
+            Certificates = certificates.Select(ConvertToPemString).ToList();
+            Type = "#Microsoft.Media.MediaGraphPemCertificateList";
+        }
+
+        private static string ConvertToPemString(X509Certificate2 certificate)
+        {
+            return new StringBuilder()
+                .AppendLine(PemCertificateHeader)
+
+                // Ideally this would be the following, but it is not supported in earlier versions of .NET
+                // .AppendLine(Convert.ToBase64String(certificate.Export(X509ContentType.Cert), Base64FormattingOptions.InsertLineBreaks))
+                .AppendLine(ToFormattedBase64String(certificate.Export(X509ContentType.Cert)))
+                .AppendLine(PemCertificateFooter)
+                .ToString();
+        }
+
+        private static string ToFormattedBase64String(byte[] data)
+        {
+            // Insert line breaks every 76 characters to match the implementation in netstandard2.0
+            // https://docs.microsoft.com/en-us/dotnet/api/system.convert.tobase64string?view=netcore-3.1#System_Convert_ToBase64String_System_Byte___System_Base64FormattingOptions_
```

Comment:
Why don't you just call that method since we target netstandard2.0? We also have a PEM helper in Azure.Core I added you could use that might help here.

Reply:
Another developer added this file so I will send him this feedback and get it updated it for GA